src/mono/mono/mini/aot-compiler.c

```diff
@@ -13047,6 +13132,20 @@ collect_methods (MonoAotCompile *acfg)
 		method = mono_get_method_checked (acfg->image, token, NULL, NULL, error);
 		report_loader_error (acfg, error, TRUE, "Failed to load method token 0x%x due to %s\n", i, mono_error_get_message (error));
 
+		if (mono_aot_mode_is_full (&acfg->aot_opts) && !mono_method_metadata_has_header (method)) {
+			// if the method has no body, see if it's an [UnsafeAccessor] method and replace it by the wrapper.
+			char *member_name = NULL;
+			int accessor_kind = -1;
+			if (mono_method_get_unsafe_accessor_attr_data (method, &accessor_kind, &member_name, error)) {
+				MonoMethod *wrapper = mono_marshal_get_unsafe_accessor_wrapper (method, (MonoUnsafeAccessorKind)accessor_kind, member_name);
```

Comment:
I wonder why inflation is not needed here? i.e. calling `replace_generated_method` instead.

Reply:
This part is iterating over the method definitions in the `MONO_TABLE_METHOD` table.  None of them are inflated.